app/src/main/java/com/ncnf/repositories/UserRepository.java

```diff
@@ -94,4 +96,12 @@ public UserRepository(FirebaseDatabase db) {
     public CompletableFuture<GeoPoint> getUserPosition(String uuid){
         return this.db.getField(USERS_COLLECTION_KEY + uuid, USER_LOCATION_KEY);
     }
+
+    public CompletableFuture<String> getUserFullName(String uuid){
+        return this.db.getField(USERS_COLLECTION_KEY + uuid, FULL_NAME_KEY);
+    }
+
+    public CompletableFuture<String> getUserUsername(String uuid){
+        return this.db.getField(USERS_COLLECTION_KEY + uuid, USERNAME_KEY);
+    }
```

Comment:
Don't forget to also test this new methods.

Reply:
+1